integration-tests/ahm/src/checks.rs

```diff
@@ -43,10 +43,10 @@ impl RcMigrationCheck for SanityChecks {
 	}
 
 	fn post_check(_: Self::RcPrePayload) {
-		assert_eq!(
-			pallet_rc_migrator::RcMigrationStage::<RcRuntime>::get(),
-			pallet_rc_migrator::MigrationStage::MigrationDone
-		);
+		// assert_eq!(
```

Comment:
yeah this one was stuck at `MigrationSignal` or similar, probably because our snapshot, for whatever reason, is still not taken at the right time :(

Reply:
I tested locally with the _snaps_ from [this run](https://github.com/paritytech/ahm-dryrun/actions/runs/17954044356/job/51060475655) and works as expected.